tests/test_xarray.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+from affine import Affine
+from dask import delayed
+
+import geowombat as gw
+from geowombat.backends.xarray_ import delayed_to_xarray
+from geowombat.data import (
+    l3b_s2b_00390821jxn0l2a_20210319_20220730_c01,
+    l8_224077_20200518_B2,
+)
+
+
+def scale(data: xr.DataArray) -> xr.DataArray:
+    return data * 1e-4
+
+
+def expand_time(dataset):
+    """`open_mfdataset` preprocess function."""
+    attrs = dataset.attrs.copy()
+    attrs['transform'] = Affine(*attrs['transform'])
+    attrs['res'] = tuple(attrs['res'])
+    # Get the date
+    file_date = datetime.strptime(
+        Path(dataset.encoding['source']).stem.split('_')[3], '%Y%m%d'
+    )
+    darray = (
+        dataset.to_array()
+        .rename({'variable': 'band'})
+        .assign_coords(time=file_date, y=dataset.y, x=dataset.x)
+        .expand_dims('time')
+        .transpose('time', 'band', 'y', 'x')
+        .where(lambda x: x != x.nodatavals[0])
+    )
+
+    return darray.map_blocks(scale, template=darray).assign_attrs(**attrs)
+
+
+class TestXarray(unittest.TestCase):
+    def test_open(self):

```

Comment:
@mmann1123 This test doesn't use `geowombat`, but it does test the `xarray` open method that caused me issues in previous `dask`/`xarray` version pairs.